readme.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://github.com/terkelg/prompts/raw/master/prompts.png" alt="Prompts" width="500" />
+  <img src="./prompts.png" alt="Prompts" width="500" />
```

Comment:
This change: 

1. Allows offline local rendering.
1. Improves portability if the repo were ever to move.
1. Makes the markdown more concise.

Reply:
Nice one! How does this work with the readme shown on NPM? @lukeed you spent some time researching this right? I would be nice to have these readme updates in their own PR

Reply:
The images still work on the npm registry page. For example, see: https://www.npmjs.com/package/list-maintainers.

Reply:
I'm happy to split these changes out. I assume you only want the image href updates split out, but the docs for this feature should stay in this PR?